src/app/join/page.tsx

```
@@ -0,0 +1,225 @@
+  Title,
+} from "@mantine/core";
+import {
+  IconAlertCircle,
+  IconBrandGoogle,
+  IconCheck,
+  IconTicket,
+} from "@tabler/icons-react";
+import Image from "next/image";
+import Link from "next/link";
+import { signIn } from "next-auth/react";
+import { useSearchParams } from "next/navigation";
```

ESLint

[error] `next/navigation` import should occur before import of `next-auth/react`

(import/order)